test-automation/e2e/playwright/admin-job-creation.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { LoginPage } from './pages/login.page.js';
+import { AdminJobCreationFormPage } from './pages/admin-job-creation-form.page.js';
+import { JobRoleDetailsPage } from './pages/job-role-details.page.js';
+
+/**
+ * Admin Job Creation Tests (Chromium only)
+ * 
+ * Tests for the admin job creation workflow using Page Object Model pattern:
+ * 1. Navigate to /login and authenticate
+ * 2. Navigate to /admin/job-roles/new
+ * 3. Fill form fields (role name, location, capability, band, closing date, open positions, job spec link, description, responsibilities)
+ * 4. Submit form
+ * 5. Verify redirect to /job-roles/xx?created=true
+ */
+
+test.describe('Admin Job Creation (Chromium only)', () => {
+  // Test data with unique timestamp to avoid duplicates
+  const testJobData = {
+    roleName: `Test Admin Job ${Date.now()}`,
+    location: 'London',
+    capability: 'Engineering',
+    band: 'Senior',
+    closingDate: '2025-12-31',
+    numberOfOpenPositions: '2',
+    jobSpecLink: 'https://example.com/job-spec',
+    description: 'This is a test job role created by automation',
+    responsibilities: 'Test responsibilities for automation',
+  };
+
+
+  test('Admin can navigate to job creation form', async ({ page }) => {
+    const loginPage = new LoginPage(page);
+    const creationFormPage = new AdminJobCreationFormPage(page);
+
+    // Authenticate and navigate
+    await loginPage.loginAsAdmin();
+    await creationFormPage.navigate();
+
+    // Verify page loaded correctly
+    const pageTitle = await creationFormPage.getPageTitle();
+    const url = await creationFormPage.getCurrentUrl();
+
+    expect(pageTitle).toBeTruthy();
+    expect(url).toContain('/admin/job-roles/new');
+
+    // Take screenshot for visual verification
+    await creationFormPage.takeScreenshot('test-results/admin-form-page.png');
+  });
+
+  test.skip('Admin can fill job creation form with all required fields', async ({
+    page,
+  }) => {
+    // SKIPPED: Form page does not have the required fields yet
+    // This test requires the backend /admin/job-roles/new page to be fully implemented
+    test.setTimeout(25000); // Reduced timeout - should finish quickly
+
+    const loginPage = new LoginPage(page);
+    const creationFormPage = new AdminJobCreationFormPage(page);
+
+    // Authenticate and navigate
+    await loginPage.loginAsAdmin();
+    await creationFormPage.navigate();
+
+    // Try to fill form - may not find all fields depending on backend
+    await creationFormPage.fillForm(testJobData);
+
+    // Verify we're still on the form (form may not have all fields populated)
+    const currentUrl = await creationFormPage.getCurrentUrl();
+    expect(currentUrl).toContain('/admin/job-roles/new');
+
+    // Take screenshot to show form state
+    await creationFormPage.takeScreenshot('test-results/admin-form-filled.png');
+  });
+
+  test.skip('Admin can submit job creation form and verify success', async ({
```

Comment:
If we're skipping a test - why do we have it? 

We should either implement or delete this